src/main/java/org/apache/commons/io/input/BoundedInputStream.java

```diff
@@ -405,9 +399,9 @@ public synchronized long getCount() {
     }
 
     /**
-     * Gets the max count of bytes to read.
+     * Returns the maximum number of bytes this stream is allowed to read.
```

Comment:
In general, Commons uses the convention that a getter method "Gets something", a setter method "Sets something", and an is method "Tests something". Please do not change the style here.